src/Button/Button.scss

```diff
@@ -0,0 +1,4 @@
+.Button {
```

Comment:
Just put this in as a point to test with.. directory structure is all up in the air.